Update documentation for pagination (#1435)
google/osv.dev: docs/api/post-v1-querybatch.md

The parameters are the same as those in found [here](post-v1-query.md#parameters), but you can make multiple queries.

> Information on paginated results for querybatch requests is available [here](#pagination).

Reviewer: Minor, but better not to put links under "here" --- prefer descriptive links when possible. `See [#pagination] for information on paginated results for querybatch requests.`

> Information on paginated results for querybatch requests is available [here](#pagination).

Author: ```suggestion
[Instructions are available](#pagination) for handling pagination for querybatch requests. 
```

        }
    ]
}
```

## Pagination

> Pagination for the querybatch API works similarly to the individual query, with the distinction that not all queries in the querybatch are likely to return a `next_page_token`. 

Reviewer: ```suggestion
Pagination for the querybatch API works similarly to the individual query. However, a querybatch request may return results with a `next_page_token` for only a few of the total queries. In this situation, you will need to run additional requests  for those specific queries to see the remaining results.
```

A queryset response with paginated results will be in this form:

```json
{
  "results": [
    {
      "vulns": [
        ...
      ],
      "next_page_token": "token for query 1"
    },
    {
      "vulns": [
        ...
      ],
      "next_page_token": "token for query 2"
    },
    {
      "vulns": [
        ...
      ],
    },
    ...
  ]
}
```

> Notice that each result has a distinct "next_page_token" and that the third result does not include a `next_page_token`. This is because all of the vulnerabilities for the third query have been returned. 

Reviewer: ```suggestion
Notice that each result has a distinct "next_page_token" and that the third result does not include a `next_page_token`. This indicates that all of the vulnerabilities for the third query have been returned. 
```

> To get the next page of results, your next request should be in the form:

Reviewer: Can you specify in this sentence what differs? `To get the next page of results, your next request should specify `page_token` only for the queries that returned `next_page_token`.`

> To get the next page of results, your next request should be in the form:

Author: ```suggestion
To get the next page of results, your next request should specify `page_token` only for the queries that returned `next_page_token`. 
```
